11ty/definitions/semitic.md

```diff
@@ -1,5 +1,23 @@
 ---
 title: Semitic
 slug: semitic
-defined: false
----
\ No newline at end of file
+flag: _ _
+defined: true
+excerpt: 
+    1. Related to languages, races, and culture of people who live or come from across part of western Asia and northern Africa. There are included Arabic, Hebrew, Aramaic and Amharic.
+    2. Mainly referred to as Arabic and Hebrew
+speech: adjective
+alt_words:
+- _ _
+- _ _
+---
+
+1. Related to languages, races, and culture of people who live or come from across part of western Asia and northern Africa. There are included Arabic, Hebrew, Aramaic and Amharic.
+2. Mainly referred to as Arabic and Hebrew.
+
```

Comment:
Here, I'd add a note like this:

```
## Note
While Semitic refers to a broad spectrum of languages, races, and cultures above, it is often mistakenly only used to speak about Arabic and Hebrew.

Reply:
I would also add a section for Related Words, and add [antisemitic](https://www.selfdefined.app/definitions/antisemitism/) Because you're linking internally, you will only need to link `/definitions/antisemitism`